hack/tools/tools.go

```diff
@@ -1,5 +1,5 @@
-//go:build tools
-// +build tools
+//go:build vmop_tools
+// +build vmop_tools
```

Comment:
Out of curiosity, is there a reason behind this rename?

Reply:
See [2] in the commit message: one of golangci-lint's dependencies recently added a file with the "tools" build tag, which is fine when just building that dependency alone, but breaks when being pulled into the golangci-lint build. This will likely end up breaking various CAPI-inspired hack/tools.